example/main.go

```diff
@@ -28,7 +30,8 @@ func main() {
 	}
 }
 
-func ReceiveToServer(action *structs.Action) (*structs.Message, error) {
+
+  func ReceiveToServer(action *structs.Action) (*structs.Message, error) {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function ReceiveToServer should have comment or be unexported